.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
we don't want to ignore package-lock.json because it saves the last successful installed packages' versions. It's useful in case something went wrong with the packages we are using on npm

Reply:
fixed